charts/internal/shoot-control-plane/templates/network-policies.yaml

```diff
@@ -88,3 +88,18 @@ spec:
     - cidr: {{ (split ":" $endpoint)._0 }}/32
 {{- end }}
 {{- end }}
+---
+apiVersion: metal-stack.io/v1
+kind: ClusterwideNetworkPolicy
+metadata:
+  name: allow-to-konnectivity
+  namespace: firewall
+spec:
+  egress:
+  - to:
+{{- range $i, $ip := .Values.apiserverIPs }}
```

Comment:
If len(apiserverIPs) == 0  ?

the whole cnwp should only be created of apiserverIPs are present